framework/db/Command.php

```diff
@@ -455,11 +455,13 @@ public function insert($table, $columns)
      * @param string $table the table that new rows will be inserted into.
      * @param array $columns the column names
      * @param array $rows the rows to be batch inserted into the table
+     * @param boolean $ignore whether to excute insert ignore into, only support MySQL
+     * @param boolean $replace whether to excute `repace into` instead of `insert into` , only support MySQL
      * @return $this the command object itself
      */
-    public function batchInsert($table, $columns, $rows)
+    public function batchInsert($table, $columns, $rows, $ignore = false, $replace = false)
```

Comment:
Method signature change breaks backwards compatibility so with it the pull request could be merged into 2.1 only.